app/src/androidTest/java/co/nimblehq/compose/crypto/ui/screen/detail/DetailScreenUiTest.kt

```diff
@@ -0,0 +1,146 @@
+package co.nimblehq.compose.crypto.ui.screen.detail
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.domain.usecase.GetCoinDetailUseCase
+import co.nimblehq.compose.crypto.domain.usecase.GetCoinPricesUseCase
+import co.nimblehq.compose.crypto.extension.toFormattedString
+import co.nimblehq.compose.crypto.test.MockUtil.coinDetail
+import co.nimblehq.compose.crypto.test.MockUtil.coinPrices
+import co.nimblehq.compose.crypto.test.TestDispatchersProvider
+import co.nimblehq.compose.crypto.ui.components.chartintervals.TimeIntervals
+import co.nimblehq.compose.crypto.ui.screens.MainActivity
+import co.nimblehq.compose.crypto.ui.screens.detail.*
+import co.nimblehq.compose.crypto.ui.screens.home.FIAT_CURRENCY
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import kotlin.math.abs
+
+@ExperimentalCoroutinesApi
+class DetailScreenUiTest {
```

Comment:
To make it consistent with `HomeScreenUITest` 🙏 
```suggestion
class DetailScreenUITest {
```

Reply:
@Wadeewee Actually, `UiTest` follows our Kotlin convention 🤓